src/api/mockApi.js

```diff
@@ -0,0 +1,18 @@
+export const getVideos = () => {
+  return fetch(
+    'https://raw.githubusercontent.com/wizelineacademy/react-gist/main/capstone-project-1/mocks/youtube-videos-mock.json'
+  ).then((response) => response.json());
+};
+// export const getVideos = async (url) => {
```

Comment:
Remember, we have Git, so it's a good practice not to include commented out code.

Reply:
Commented out code and console log statements were corrected on the following [PR ](https://github.com/laulujan/react-apprenticeship-capstone1/pull/2)